tm-tool/textmapper/resources/org/textmapper/tool/templates/go_parser.ltp

```diff
@@ -390,7 +378,7 @@ func (p *Parser) skipBrokenCode(lexer *Lexer, stack []stackEntry, canRecover fun
 		}
 ${call reportConsumedNext-}
 		e = p.next.endoffset
-		p.fetchNext(lexer, stack${if self->ignoredReportTokens()}, nil${end})
+		p.fetchNext(lexer, stack)
```

Comment:
Invariant: before calling fetchNext we should flush all pending tokens.

In most places p.pending is empty because of p.next.symbol == noToken, so we don't need to do anything. Here we should flush the previous batch _before_ reportConsumedNext. How about we extract "if len(p.pending) > 0 { for {...} ... }" into a flushPending template and call it from here as well?

Reply:
I like the idea of adding a template! I didn't realize it's an option. Done.